interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +840,23 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+
+<?php if(isset($_GLOBALS['validate_pages']))  :?>
+    <?php foreach($_GLOBALS['validate_pages'] as $validation) :?>
+        <?php if($validation['page_name']==basename(__FILE__)):?>
+            <?php include_once("$srcdir/validation/validation_script.js.php");?>
+            <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/jquery-1.9.1.min.js"></script>
+            <script type="text/javascript">
+                var form = document.getElementsByName("theform");
+                form.id="theform";
+                <!--Use validation script js Validations-->
+                $("form").attr("onclick",'return submitme(<?php echo $GLOBALS['new_validate'] ? 1 : 0;?>,event,"theform",<?php echo $validation['rules'];?>)');
+            </script>
+        <?php endif;?>
+    <?php endforeach;?>
+<?php endif;?>
+
```

Comment:
I have fixed this issue now the logic is that you collect the rules from each page using a function in globals called collectValidationPageRules which receives $option_id,$title as you suggested (page name,pagepath)   . also I had to change the field size from 31 to 255 varchar hope this is enough. and I have updated the upgrade 4_2_2 to 4_3_1 file and database.sql file with the new insert